tests/unit/CMakeLists.txt

```diff
@@ -13,7 +13,8 @@ FetchContent_Declare(
 FetchContent_MakeAvailable(googletest)
 
 add_executable(greentea-tests test_kv_protocol.cpp)
-target_link_libraries(greentea-tests PRIVATE greentea::client gtest_main)
+target_compile_features(greentea-tests PUBLIC cxx_std_14)
+target_link_libraries(greentea-tests PUBLIC greentea::client_userio fake-console-io gtest_main)
```

Comment:
Was `greentea-tests` the target that failed linking in CI? Did you try in CI with listing `fake-console-io` first here?

Reply:
No, it was the example apps that failed linking. I'll spend another hour trying to figure it out, it probably was something silly like that.